content/_policy/contact._en.md

```diff
@@ -29,7 +29,7 @@ help_center_content: >-
 partner_content: >-
   ## Partner with Login.gov
 
-  Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) or [contact us](https://share.hsforms.com/16DIoo--rTU2xbNW1MShkBg3ak9e).
+  Interested in using Login.gov at your agency? Please [visit our partners website](https://partners.login.gov/) or [contact us](https://login.gov/partners/contact/).
```

Comment:
**question:** do we want to update the first link to `login.gov/partners` too? Also, I'm pretty sure there's a way in Jekyll to link to internal pages without specifying the whole URL.